test/playwright/test_ux.py

```diff
@@ -48,24 +48,23 @@ def _login_sequence(page, screenshot=False):
         grab screenshots
     """
     # Log in sequence
+    if screenshot:
+        page.screenshot(path="test-results/login.png")
```

Comment:
`login.png` and `conda-store-unauthenticated.png` are the same, maybe we can keep just one?